### DISCUSSION TITLE
bookmark table of contents

### DISCUSSION THREAD

**Discussion by harryvonstein** [rgthree/rgthree-comfy#196] [Ideas]
A way to see all your bookmarks in one place and being able to click on one from there to send you to that bookmark.

**Comment by DrJKL** [comment]
Took a stab at it: https://github.com/rgthree/rgthree-comfy/pull/198

**Comment by TFWol** [comment]
@rgthree maybe Issue should be closed since there's a list now at 
<img width="116" height="56" alt="image" src="https://github.com/user-attachments/assets/555976af-d357-4f89-931b-1cd1124600d7" />
